pkg/disk/utils.go

```
@@ -1177,7 +1177,12 @@ func getAvailableDiskCount(node *v1.Node, ecsClient cloud.ECSInterface, efloC cl
 		return count, nil
 	}
 	if lingjunNodeId != "" {
-		return getLingjunAvailableDiskCount(efloC, lingjunNodeId)
+		count, err := getLingjunAvailableDiskCount(efloC, lingjunNodeId)
+		if err != nil {
+			klog.ErrorS(err, "getAvailableDiskCount: failed to get lingjun available disk count", "lingjunID", lingjunNodeId)
```

Do we want to ignore all errors? Can we be more specific here?